src/commands/shell.js

```diff
@@ -0,0 +1,38 @@
+const { requireHerbarium } = require('../generators/utils')
+const repl = require('@herbsjs/herbs2repl')
+
+const cmd = {
+  name: 'shell',
+  description: 'Run Herbs Shell',
+  run: async toolbox => {
+    const { red } = toolbox.print.colors
+    const herbarium = requireHerbarium("shell", toolbox.filesystem.cwd())
+
+    if (!herbarium.requireAll) {
+      toolbox.print.info(`\n ${red('• Exit with error: Herbarium not found 😢')}`)
+      return
+    }
+
+    herbarium.requireAll()
+
+    async function main(injection) {
+      const usecases = Array.from(herbarium.usecases.all).map(([_, item]) =>
+          ({ usecase: item.usecase(injection), id: item.id, tags: { group: item.group || "useCase" } }))
```

Comment:
`"useCase"` is not user friendly....
you can change: `{ group: item.group || "useCase" }`
to: `{ group: item.group || "Others" }`